app/electron/plugin-management.ts

```diff
@@ -52,6 +52,22 @@ interface PluginData {
   artifacthubVersion: string;
 }
 
+/**
+ * Move directories from currentPath to newPath by copying.
```

Comment:
I think a merge issue here too. I don't think this should be in this PR.

Reply:
Thanks, removed these changes (I noticed a couple of other ones as well that should not have been in there) and rebased against main.